### DISCUSSION TITLE
Hey uhh simple question but how do I build this app?

### DISCUSSION THREAD

**Discussion by thickmelon** [jeffvli/feishin#690] [Q&A]
Linux user here. Do I just cd into its directory and just execute the feishin file..? I tried to build it into a deb file with alien, dpkg, none of which worked...Do I just keep using the feishin file in the folder?

**Comment by jeffvli** [comment]
Just `npm run install --legacy-peer-deps` and then `npm run package`. For linux, it will build an appimage inside the `/release/build` folder.
